__tests__/api.test.js

```diff
@@ -93,6 +93,98 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("should respond with 200 status", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("should respond with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        expect(body).toHaveProperty("articles", expect.any(Array));
+      });
+  });
+  test("array should contain object with correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(body).toHaveProperty("articles", expect.any(Array));
+        expect(typeof articles).toBe("object");
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(String));
+        });
+        expect(articles.length).toEqual(13);
+      });
+  });
+  test("should be ordered by date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should return 200 status", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("should respond with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        expect(body).toHaveProperty("comments", expect.any(Array));
+      });
+  });
```

Comment:
This test isnt testing everything that is mentioned in the description. This test is simply asserting that there is an array attached to the key "comments", but you are not testing that these comments belong to the given article_id, which is a good test and should be included here or in the test below (if you want to add this to the test below then this test description will need to be amended).